python_modules/libraries/dagster-fivetran/dagster_fivetran/resources.py

```diff
@@ -613,6 +666,13 @@ class FivetranWorkspace(ConfigurableResource):
         default=0.25,
         description="Time (in seconds) to wait between each request retry.",
     )
+    disable_schedule_on_trigger: bool = Field(
+        default=True,
+        description=(
+            "Specifies if you would like any connector that is sync'd using this "  # TODO: update description
```

Comment:
nit; remove todo

Reply:
Done in [5a8ea97](https://github.com/dagster-io/dagster/pull/25911/commits/5a8ea977ec6888887fdc54c5b36ca2adbfa7796a)